---
layout: single
---

<p>I am a PhD student at <a href="http://cbio.ensmp.fr/">Center for computational biology</a> which is a joint research unit involving <a href="http://www.mines-paristech.eu/">Mines ParisTech</a>, <a href="http://www.institut-curie.org/?prehome=0/">Institut Curie</a> and <a href="http://english.inserm.fr/">INSERM</a>. The center is dedicated to epidemiology, bioinformatics and computational systems biology of cancer. I am co-supervised by <a href="http://cazencott.info/">Chloé-Agathe Azencott</a> and <a href="https://www.linkedin.com/in/veronique-stoven-b2a74815">Véronique Stoven</a>. </p>

<h2>Research interests</h2>

<p>The big question of my thesis is the <a href="wisdom/notes/bio/missing_heritability.html">missing heritability problem]</a>, the discordance between the estimated genetic determination of a phenotype, and the actual genetic factors we find in association with it. The community has come up with several hypothesis behind this problem, and during my thesis I intend on exploring two of them: <a href="wisdom/notes/bio/epistasis.html">epistasis</a> and the high p, low n curse. To do so, I am carrying out <a href="wisdom/notes/bio/gwas.html">GWAS</a> which include a priori knowledge, namely biological networks that give a context to every SNP. This approach allows for context-dependent statistics, where we can fine tune the significance threshold for each individual SNP. Currently we are applying these approaches to <a href="wisdom/notes/bio/familial_brca.html">familial breast cancer</a>.</p>

<h2>Short bio</h2>

Biotechnology. Biochemistry. Bioinformatics. Research.
